isbn_updater/src/comp.php

```diff
@@ -0,0 +1,352 @@
+<?php
+
+print("</br><a href='isbn_updater.html'>Back to ISBN Updater tool</a>");
+
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
+
+echo "<p>Starting</p>";
+
+//*****************GRAB_INPUT_DATA**********
+
+$uploaddir = '../uploads/';
+$uploadfile = $uploaddir . basename($_FILES['userfile']['name']);
+
+echo '<pre>';
+if (move_uploaded_file($_FILES['userfile']['tmp_name'], $uploadfile)) {
+	echo "File is valid, and was successfully uploaded.\n";
+} else {
+	echo "File is invalid, and failed to upload - Please try again. -\n";
```

Comment:
```suggestion
	echo "File failed to upload.\n";
```

Improving the user message: There has been no validation at this point, only a failed file upload.